index.js

```diff
@@ -51,6 +51,13 @@ module.exports = [...compat.extends('airbnb-base'),
                 avoidEscape: true,
                 allowTemplateLiterals: true,
             }],
+            // It's ok to use functions before they are defined.
+            'no-use-before-define': ['error', {
+                'functions': false,
+                'classes': true,
+                'variables': true,
```

Comment:
Unfortunately, since we prefer arrow functions like `const f = () => {...}` the important change is **'variables': false**

Note:
This should be alright since something like 
```
a = 10;
let a = 0;
``` 
will raise the error anyway (from ts, not only eslint).